tests/Aspire.Hosting.Tests/ProjectResourceTests.cs

```diff
@@ -14,6 +14,47 @@ namespace Aspire.Hosting.Tests;
 
 public class ProjectResourceTests
 {
+    [Fact]
+    public async Task AddProjectWithInvalidLaunchSettingsShouldThrowSpecificError()
+    {
+        var projectDetails = await PrepareProjectWithMalformedLaunchSettingsAsync();
+
+        var ex = Assert.Throws<DistributedApplicationException>(() =>
+        {
+            var appBuilder = CreateBuilder();
```

Comment:
```suggestion
        var appBuilder = CreateBuilder();
        var ex = Assert.Throws<DistributedApplicationException>(() =>
        {
```

(nit) I like to keep the action inside the Assert.Throws as minimal as possible, so it is obvious what line is throwing the exception.